packages/connectivity/src/scp-cf/cache.ts

```diff
@@ -48,9 +38,9 @@ export class Cache<T> implements CacheInterface<T> {
    * Default validity period for each entry in cache.
    * If `undefined`, all cached entries will be valid indefinitely.
    */
-  private defaultValidityTime: DateInputObject | undefined;
+  private defaultValidityTime: number | undefined;
```

Comment:
```suggestion
  private defaultValidityTimeInMs: number | undefined;
```
[req] In my opinion, it is very confusing to just have `number` as a type and not state what unit of measurement is intended

Reply:
thank you for your comment. that's right. I added `InMs`